pangeo_forge_orchestrator/commands/base.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from traitlets import List, Unicode
+from traitlets.config import Application
+
+# Common aliases we want to support in *all* commands
+# The key is what the commandline argument should be, and the
+# value is the traitlet config it will be translated to
+common_aliases = {
+    "log-level": "Application.log_level",
+    "f": "BaseCommand.config_file",
+    "config": "BaseCommand.config_file",
+}
+
+
+class BaseCommand(Application):
+    """
+    Base Application for all our subcommands.
+    Provides common traitlets everyone needs.
+    Do not directly instantiate!
```

Comment:
@alxmrs, like this

```python
from abc import ABC
...

class BaseCommand(Application, ABC):
    ...
```
?

Reply:
Yup, that looks good :)

Reply:
You can also annotate specific methods that need to be overridden with:

```python
    @abc.abstractmethod
    def execute(self) -> None:
        pass
```

Reply:
Thanks for the review, @alxmrs. I've actually decided to supersede this PR with #204, so going to close this. But fwiw `ABC` is making an appearance in #204 (though in a different context, because I'm dropping this BaseCommand for simplicity).